doc/CX10Blue.txt

```diff
@@ -131,7 +131,7 @@ XOR with a fixed whitening sequence:
 Packets are GFSK modulated at a rate of 1 Mbit/second and a frequency deviation
 of approximately +/- 340 kHz.  Zero bits are represented by the lower frequency
 deviation while one bits are represented by the higher deviation.  Each packet
-has a duration of 245 microseconds and is preceded by an modulated carrier
+has a duration of 245 microseconds and is preceded by an not modulated carrier
```

Comment:
_a_ not modulated... it made a lot more sense before (an unmodulated)...

Reply:
I'm not an English native, but unmodulated sounds fine to me.

Reply:
Indeed, unmodulated is fine.